src/overlays/actors/ovl_Bg_Haka_Tubo/z_bg_haka_tubo.h

```diff
@@ -11,7 +11,10 @@ typedef void (*BgHakaTuboActionFunc)(struct BgHakaTubo*, GlobalContext*);
 typedef struct BgHakaTubo {
     /* 0x0000 */ DynaPolyActor dyna;
     /* 0x0164 */ BgHakaTuboActionFunc actionFunc;
-    /* 0x0168 */ char unk_168[0x9C];
+    /* 0x0168 */ s16 dropTimer;
+    /* 0x0168 */ s16 fireScroll;
+    /* 0x016C */ ColliderCylinder collider;
+    /* 0x016C */ ColliderCylinder colliderFlames;
```

Comment:
I suggest these names for colliders, or maybe `mainCollider` instead of `potCollider`.
```suggestion
    /* 0x016C */ ColliderCylinder potCollider;
    /* 0x016C */ ColliderCylinder flamesCollider;
```